static/app/components/onboarding/gettingStartedDoc/step.tsx

```diff
@@ -0,0 +1,37 @@
+import beautify from 'js-beautify';
+
+import {CodeSnippet} from 'sentry/components/codeSnippet';
+import {t} from 'sentry/locale';
+
+export enum StepType {
+  INSTALL = 'install',
+  CONFIGURE = 'configure',
+  VERIFY = 'verify',
+}
+
+export const StepTitle = {
+  [StepType.INSTALL]: t('Install'),
+  [StepType.CONFIGURE]: t('Configure'),
+  [StepType.VERIFY]: t('Verify'),
+};
+
+export type StepProps = {
+  code: string;
+  description: React.ReactNode;
+  language: string;
+  type: StepType;
+};
+
+export function Step({type, description, language, code}: StepProps) {
+  return (
+    <div>
+      <h4>{StepTitle[type]}</h4>
+      <p>{description}</p>
+      <CodeSnippet dark language={language}>
+        {language === 'javascript'
+          ? beautify.js(code, {indent_size: 2, e4x: true})
+          : beautify.html(code, {indent_size: 2})}
```

Comment:
This won't work for everything in the future right

Reply:
Why do we actually need this, can't we just format our code in our examples already?

Reply:
this is the library we are using to format our code and javascript is treated differently. We'll probably have to revise this when migrating other documents, but for now, we're only doing this for javascript.

We need this as only our format does not render nicelly

Reply:
I'm confused, aren't we hard coding our example code? Why do we need a formatter?

Reply:
yes we are hard coding but in string format and for it to have nice indentations, etc we need the beautify library

Reply:
Why can't we format it in our hard coding?

Reply:
probably we can but it's annoying and since we are already using this library elsewhere...why not reuse it?

Reply:
Sure, but we're probably not going to want to include a bunch of other formatting libraries for all the other languages right

Reply:
yes, we will have to think about that soon... will add a note to revise it